src/InteractiveWindow/Editor/InteractiveWindow.cs

```diff
@@ -734,6 +737,55 @@ private void CutOrDelete(IEnumerable<SnapshotSpan> projectionSpans, bool isCut)
             }
         }
 
+        /// <summary>
+        /// Copy the entire selection to the clipboard for RTF format and
+        /// copy the selection minus any prompt text for other formats.
+        /// That allows paste into code editors of just the code and
+        /// paste of the entire content for editors that support RTF.
+        /// </summary>
+        private void CopySelection()
+        {
+            var spans = _textView.Selection.SelectedSpans;
+            var text = spans.Aggregate(new StringBuilder(), GetTextWithoutPrompts, b => b.ToString());
```

Comment:
CopySelection is only called once for a user action so the allocation won't be an issue.